irods/test/helpers.py

```diff
@@ -290,9 +290,14 @@ def irods_session_host_local (sess):
 def enableLogging(logger, handlerType, args, level_ = logging.INFO):
     """Context manager for temporarily enabling a logger. For debug or test.
 
-    Usage Example -
-    with irods.parallel.enableLogging(logging.FileHandler,('/tmp/logfile.txt',)):
-        # parallel put/get code here
+    Usage Example
+    -------------
+    Dump INFO and higher priority logging messages from irods.parallel
+    module to file /tmp/logfile.txt:
+
+    with irods.test.helpers.enableLogging(logging.getLogger('irods.parallel'),
+                                          logging.FileHandler,('/tmp/logfile.txt',)):
```

Comment:
What happens if there's no /tmp directory?
Can users pick a different location?

Reply:
Right, just an example but - point taken. Examples should also be portable : )

Reply:
I don't necessarily think the example needs to be portable. However, looking at the example made me wonder if the things I asked about were possible.

I think changing the example to include comments explaining or demonstrating what can be changed would help users.

Reply:
yes, let's use the os-independent temporaryFile thinger and/or additional examples.